### PR TITLE
Added Molex 502250 generator, and various tidy-ups

### PR DESCRIPTION
**Pull request by jhol**

### CODE REVIEW

**Comment by easyw** [comment]
hi @jhol 
running the script I'm getting this
molex
Am I missing something?

File: cadquery/FCAD_script_generator/molex/cq_models/conn_molex_502250.py [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+    for p in [anchor_pins[1]] + odd_pins + even_pins:
+        pins = pins.union(p)
+    return pins.rotate((0, 0, 0), (0, 0, 1), -90)
+
+
+def generate_body(body_length):
+    slot_length = body_length - 0.6*2
+
+    body_outline = cq.Workplane("XY").polyline([
+        (0, 0.1),
```
Comment (poeschlr) [inline cadquery/FCAD_script_generator/molex/cq_models/conn_molex_502250.py]: This line might be the trouble maker for the problem encountered by @easyw

Comment (poeschlr) [reply cadquery/FCAD_script_generator/molex/cq_models/conn_molex_502250.py]: It seems in cadquerry 0.5.2 the polygone stuff always adds a point at 0,0.
the solution would be to first use moveTo.

If i change the lines to:
```
 body_outline = cq.Workplane("XY")\
        .moveTo(0, 0.1)\
        .polyline([
        #(0, 0.1),
        (body_length/2-0.745, 0.1),
```

it does work.

File: cadquery/FCAD_script_generator/molex/cq_models/conn_molex_502250.py [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+            .moveTo(-slot_length/2, 0.26)
+            .rect(slot_length, 0.28, centered=False)
+            .extrude(-series_params.body_width)
+        ).translate((0, -series_params.body_y_offset, 0.02)).rotate((0, 0, 0), (0, 0, 1), -90)
+
+
+def generate_latch(body_length):
+    bar_length = body_length - 1.58
+
+    return cq.Workplane("YZ").polyline([
```
Comment (poeschlr) [inline cadquery/FCAD_script_generator/molex/cq_models/conn_molex_502250.py]: This part works if i change it to:
```
return cq.Workplane("YZ")\
        .moveTo(1.41, 0.89)\
        .polyline([
            #(1.41, 0.89), 
            (0, 0.89), 
            (0, 0.4), 
            (1.4-0.65, 0.4),
            (0.75, 0.15), 
            (1.01, 0.15), 
            (1.41, 0.585),
            ]) \
        .close() \
```

**Comment by jhol** [comment]
Try now - the problems you mentioned should now be fixed.

**Comment by easyw** [comment]
thx, now it is working 😸

**Pull request merged**